client/state/posts/test/selectors.js

```diff
@@ -364,4 +360,30 @@ describe( 'selectors', () => {
 			expect( isRequesting ).to.be.false;
 		} );
 	} );
+
+	describe( '#getPostsBySiteIdAndPostId()', () => {
+		it( 'should return global post id by siteId and postId', () => {
+			const postsBySiteIdAnPostId = getPostsBySiteIdAndPostId( {
+				posts: {
+					items: {
+						'3d097cb7c5473c169bba0eb8e3c6cb64': { ID: 841, site_ID: 2916284, global_ID: '3d097cb7c5473c169bba0eb8e3c6cb64', title: 'Hello World' },
+						'6c831c187ffef321eb43a67761a525a3': { ID: 413, site_ID: 2916284, global_ID: '6c831c187ffef321eb43a67761a525a3', title: 'Ribs & Chicken' },
+						'9bba0eb8e3c6cb643d097cb7c5473c16': { ID: 222, site_ID: 1111111, global_ID: '9bba0eb8e3c6cb643d097cb7c5473c16', title: 'Steak & Eggs' }
```

Comment:
As random as it seems, `1111111` correlates to a user's personal blog. I usually cycle between `2916284`, `77203074`, and `77203199` as test site IDs.